contrib/debian/zfsutils-linux.zfs-share.init

```diff
@@ -0,0 +1 @@
+../etc/init.d/zfs-share
```

Comment:
Can we get rid of all this zfsutils-linux stuff? We don't build a zfsutils-linux package as far as I can tell.